rust/src/wallet.rs

```diff
@@ -119,9 +119,11 @@ mod tests {
     #[test]
     fn wallet_support_for_different_bitcoin_networks() {
         let temp = temp_dir();
-        wallet::init_wallet(Network::Mainnet, &temp).expect("wallet to be initialized");
-        wallet::init_wallet(Network::Testnet, &temp).expect("wallet to be initialized");
-        wallet::init_wallet(Network::Regtest, &temp)
+        wallet::init_wallet(Network::Mainnet, &temp.join("mainnet"))
+            .expect("wallet to be initialized");
+        wallet::init_wallet(Network::Testnet, &temp.join("testnet"))
+            .expect("wallet to be initialized");
+        wallet::init_wallet(Network::Regtest, &temp.join("regtest"))
```

Comment:
❓ So these tests were broken?

Reply:
yup. got broken yesterday, seems like it :)

Reply:
lol, they're still broken :D fixing up!